Metabase fails to launch in Chinese

```
java -jar metabase.jar 
```

and i get the error
```
09-20 11:00:40 INFO metabase.util :: 正在加载Metabase……
09-20 11:00:42 INFO util.encryption :: 已存储的证书加密方式已经为这个Metabase事例启用 🔐 
For more information, see https://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest
Exception in thread "main" java.lang.ExceptionInInitializerError
....
Caused by: java.lang.IllegalArgumentException: can't parse argument number: Login
	at java.text.MessageFormat.makeFormat(MessageFormat.java:1429)
	at java.text.MessageFormat.applyPattern(MessageFormat.java:479)
	at java.text.MessageFormat.<init>(MessageFormat.java:380)
	at puppetlabs.i18n.core$fmt.invoke(core.clj:250)
	at puppetlabs.i18n.core$translate.doInvoke(core.clj:257)
	at clojure.lang.RestFn.invoke(RestFn.java:445)
	at metabase.integrations.ldap$fn__48754.invokeStatic(ldap.clj:48)
	at metabase.integrations.ldap$fn__48754.invoke(ldap.clj:47)
	at metabase.integrations.ldap__init.load(Unknown Source)
	at metabase.integrations.ldap__init.<clinit>(Unknown Source)
	... 126 more
Caused by: java.lang.NumberFormatException: For input string: "Login"
```

what should i do?

--- a/locales/zh.po
+++ b/locales/zh.po
@@ -8006,7 +8006,7 @@ msgstr "用户搜索基础.（将采用递归检索）"
 
 #: src/metabase/integrations/ldap.clj
 msgid "User lookup filter, the placeholder '{login}' will be replaced by the user supplied login."
-msgstr "用户查找过滤器，占位符“{Login}”将被用户提供的登录替换"
+msgstr "用户查找过滤器，占位符'{login}'将被用户提供的登录替换"
 
 #: src/metabase/integrations/ldap.clj
 msgid "Attribute to use for the user's email. (usually ''mail'', ''email'' or ''userPrincipalName'')"

